### PR TITLE
ELEMENTS-1363: show value in readonly html editor (3.0.x)

### PR DESCRIPTION
**Pull request by Gabez0r**
Backport of https://github.com/nuxeo/nuxeo-elements/pull/412 to 3.0.x.

### CODE REVIEW

File: ui/widgets/nuxeo-html-editor.js [hunk 1]
```diff
@@ -182,7 +182,7 @@ import './quill/quill-snow.js';
     _valueChanged() {
       if (this._editor && !this._internalChange) {
         const delta = this._editor.clipboard.convert({ html: this.value });
-        this._editor.setContents(delta, Quill.sources.USER);
+        this._editor.setContents(delta, this.readOnly ? Quill.sources.SILENT : Quill.sources.USER);
```
Comment (richardsd) [inline ui/widgets/nuxeo-html-editor.js:185]: Just out of curiosity, what does this `Quill.sources.SILENT` does? It is like setting the quill editor in read only?

**Review by richardsd — APPROVED** [review]
Btw, should you run the legacy QA pipeline on this one?

**Review by mnixo — APPROVED** [review]

File: ui/widgets/nuxeo-html-editor.js (hunk 1 above)
Comment (Gabez0r) [reply ui/widgets/nuxeo-html-editor.js:185]: There are three types of sources: `USER`, `API` and `SILENT`. According to the [docs](https://quilljs.com/docs/api/#setcontents), setContent (and most other methods), won't work if source is `USER` and the editor is in read-only mode. I was not sure if we should use `API` or `SILENT` here (as both work), but went with `SILENT` for consistency with the next line of code.

**Comment by Gabez0r** [comment]
> Btw, should you run the legacy QA pipeline on this one?

@richardsd , yes the pipelines are already running for both master and 3.0.x. Will post a comment once they run successfully. 🤞

**Comment by Gabez0r** [comment]
Pipeline check 🔵 : https://qa.nuxeo.org/jenkins/job/2021/job/web-ui-pipeline/51/

**Pull request merged**